prow/config.yaml

```diff
@@ -462,6 +462,8 @@ tide:
         dashboard:
           skip-unknown-contexts: true
           from-branch-protection: true
+  batch_size_limit:
```

Comment:
Won't this break the `checkconfig` on unknown fields?

Reply:
It should have... I was expecting to need to `/override`. It looks like it passed with a cached test result instead of running the test?
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/test-infra/14102/pull-test-infra-bazel/1167131928000729089

Reply:
lol

Reply:
Nevermind, thats a different PR. It ran on this PR: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/test-infra/14139/pull-test-infra-bazel/1167196685286248449

We build and run checkconfig with the PR changes as part of `//hack:verify-config`: https://github.com/kubernetes/test-infra/blob/f0114293799b2f2b5b29fbf19f44d3e10882234d/hack/BUILD.bazel#L359

Reply:
Well that leaves me stumped